generators/server/templates/src/main/resources/config/_bootstrap.yml

```diff
@@ -0,0 +1,10 @@
+# ===================================================================
+# Spring Cloud Config bootstrap configuration
+# ===================================================================
+
+spring:
+    application:
+        name: <%= baseName %>
+    cloud:
+        config:
+            uri: http://localhost:8761/config
```

Comment:
According to the doc, we could have boostrap-dev.yml

http://projects.spring.io/spring-cloud/spring-cloud.html#customizing-bootstrap-properties